torch/onnx/utils.py

```diff
@@ -801,6 +805,18 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
     return torch_out
 
 
+def _replace_unique_output_ids(graph, params):
```

Comment:
Please name this `_apply_friendly_debug_names` or something like that.
(The current name is bad because this doesn't actually do anything to output IDs)